### PR TITLE
SLSA v1.0: shift emphasis to expectations + verification in levels.md

### PR DESCRIPTION
**Pull request by MarkLodato**
**Preview:** https://deploy-preview-503--slsa.netlify.app/spec/v1.0/levels

(As discussed in #130 and the Specification SIG meeting)

Previously the specification only required the publication of provenance but did not say anything about its verification. The latter is what actually detects or prevents attacks, so this was a big gap. Futhermore, the previous "scripted build" requirement did not have a clear reason why it was included.

Now there is explicit language around:

- Defining an expectation of how the package should be built, replacing the previous "scripted build" requirement.
- Verifying that the provenance meets expectations.

NOTE: This PR only changes levels.md. A future commit will make an equivalent change to the rest of the spec, e.g. requirements.md.

Related issues (not fully fixed): #46, #130, #371

### CODE REVIEW

File: docs/spec/v1.0/levels.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ tracks without invaliding previous levels.
 | Track/Level | Requirements | Benefits |
 | ----------- | ------------ | -------- |
 | [Build L0]  | (none)       | (n/a)    |
-| [Build L1]  | Attestation showing how the package was built | Documentation, inventorying |
+| [Build L1]  | Attestation showing that the package was built as expected/documented | Documentation, mistake prevention, inventorying |
```
Comment (joshuagl) [inline docs/spec/v1.0/levels.md:52]: Can we just say "as expected", given we use that term in the track summary and the L1 requirements?

File: docs/spec/v1.0/levels.md [hunk 2]
```diff
@@ -114,8 +118,8 @@ n/a
 <dl class="as-table">
 <dt>Summary<dd>
 
-Package has a provenance attestation showing how it was built, though without
-tamper protection.
+Package has a provenance attestation showing that it was built as
+expected/documented, though without tamper protection.
```
Comment (joshuagl) [inline docs/spec/v1.0/levels.md:122]: same here, can we just use "as expected"?

**Review by joshuagl — COMMENTED** [review]
Great to see this being fleshed out. I made suggestions inline to consistently use "as expected" rather than "as documented/expected".

**Comment by MarkLodato** [comment]
Thanks. I updated the language as suggested and rebased onto the latest version of #502.

**Review by joshuagl — APPROVED** [review]
LGTM, thanks. Do you think we should provide guidance on how to verify provenance meets expectations (and is signed) for users of the SLSA provenance format?

**Comment by MarkLodato** [comment]
> LGTM, thanks. Do you think we should provide guidance on how to verify provenance meets expectations (and is signed) for users of the SLSA provenance format?

Yes. I assume that would be a necessary part of #46. This PR here is only a first step.

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Review by abacchi — APPROVED** [review]

**Pull request merged**